css/index.css

```diff
@@ -1,3 +1,432 @@
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+
+.container {
+	margin: 0 auto;
+	width: 876px;
+}
+
+.nav {
+	display: flex;
+	width: 100%;
+    justify-content: space-between;
+    margin-top: 30px;
+}
+
+.nav-no-logo {
+	display: flex;
+    width: 100%;
+    justify-content: space-between;
+    margin-top: 30px;
+    margin-right: 50px;
+}
+
+.nav a {
+	text-align: left;
+    font-size: 16px;
+    align-self: center;
+	color: gray;
+    text-decoration: none;
+}
+
+
+.main-header {
+	display: flex;
+	text-align: center;
+	margin-top: 75px;
+	width: 100%;
+	margin-bottom: 75px;
+	flex-wrap: wrap;
+	flex-direction: row;
+}
+
+.main-header .top-image-container {
+    align-items: flex-end;
+	width: 49%;
+}
+
+.main-header img {
+	display: flex;
+    margin-left: 75px;
+    align-self: flex-end;
+
+}
+
+.main-header .big-text {
+	display: flex;
+	justify-content: flex-end;
+    width: 49%;
+    margin-bottom: 30px;
+    flex-wrap: wrap;
+}
+
+.main-header h1 {
+    font-size: 74px;
+    display: flex;
+    align-self: flex-end;
+}
+
+.main-header .get-started-container {
+	order: 2;
+	justify-content: center;
+	display: flex;
+    width:50%;
+    margin-right: 50px;
+    margin-bottom: 15px;
+}
+
+.main-header p {
+	border: 1px solid black;
+    padding: 5px 45px;
+    align-self: center;
+    justify-content: flex-start;
+}
+
+
+
+.main-content1 {
+	margin: 20px 0;
+	display: flex;
+	width: 100%;
+}
+
+
+.main-content1 h2 {
+    margin: 10px 0;
+    
+}
+
+.main-content1 p {
+	margin: 10px 0;
+	width: 100%;
+}
+
+/* Don't need styles for these, but leaving them just in case */
+
+/* .main-content1 .features {
+
+}
+
+.main-content1 .about {
+	
+} */
+
+.middle-image img{
+	margin: 20px auto;
+	width: 100%;
+}
+
+.main-content2 {
```

Comment:
You can just set the width to 30%, keep in mind that the children will inherit the parents (container) width.